appium-dotnet-driver/Appium/Android/AndroidDriver.cs

```diff
@@ -255,5 +255,44 @@ public ConnectionType ConnectionType
                 }
             }
         }
+
+        /// <inheritdoc />
+        public void SetClipboard(ClipboardContentType contentType, string base64Content)
+        {
+            switch (contentType)
+            {
+                case ClipboardContentType.Image:
+                case ClipboardContentType.Url:
+                    throw new NotImplementedException($"Android only supports contentType: {nameof(ClipboardContentType.PlainText)}");
+                case ClipboardContentType.PlainText:
+                    AppiumCommandExecutionHelper.SetClipboard(this, contentType, base64Content);
```

Comment:
should it call SetClipboardText?

Reply:
I had it call setClipboard in an earlier commit. I think a test was failing, which I then fixed. Let me check

Reply:
Not sure, but I think other clients allow users to receive a base64 string for PlainText. I left the GetClipboard and SetClipboard methods to do this, with the Set/GetClipboardText methods to return the decoded string. What do you think?